src/providers/BasicPoolsProvider.tsx

```diff
@@ -33,54 +32,61 @@ import META_SWAP_ABI from "../constants/abis/metaSwap.json"
 import { MetaSwap } from "../../types/ethers-contracts/MetaSwap"
 import { PoolRegistry } from "../../types/ethers-contracts/PoolRegistry"
 import { Web3Provider } from "@ethersproject/providers"
+import { Zero } from "@ethersproject/constants"
 import { getMigrationData } from "../utils/migrations"
-import { parseBytes32String } from "ethers/lib/utils"
+import { parseBytes32String } from "@ethersproject/strings"
 import { useActiveWeb3React } from "../hooks"
 import { useSelector } from "react-redux"
 
 type SharedSwapData = {
-  poolAddress: string
-  lpToken: string
-  typeOfAsset: PoolTypes
-  poolName: string
-  targetAddress: string | null
-  tokens: string[]
-  isSaddleApproved: boolean
-  isRemoved: boolean
-  isGuarded: boolean
-  isPaused: boolean
-  virtualPrice: BigNumber
   adminFee: BigNumber
-  swapFee: BigNumber
   aParameter: BigNumber
-  tokenBalances: BigNumber[]
+  isGuarded: boolean
+  isPaused: boolean
+  isRemoved: boolean
+  isSaddleApproved: boolean
+  isSynthetic: boolean
+  lpToken: string
   lpTokenSupply: BigNumber
   miniChefRewardsPid: number | null
-  isSynthetic: boolean
-  underlyingTokens: string[]
-  basePoolAddress: string
-  metaSwapDepositAddress: string
-  underlyingTokenBalances: BigNumber[]
+  poolAddress: string
+  poolName: string
+  swapFee: BigNumber
+  targetAddress: string | null // the address from which a pool was cloned
+  tokenBalances: BigNumber[]
+  tokens: string[]
+  typeOfAsset: PoolTypes
+  virtualPrice: BigNumber
 }
 
 type MetaSwapInfo = SharedSwapData & {
```

Comment:
tbh I liked keeping `SharedSwapData` as only the types that were shared, so I changed it back